trustme/_cli.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+
+import argparse
+import os
+import trustme
+import sys
+
+# Python 2/3 annoyingness
+try:
+    unicode
+except NameError:  # pragma: no cover
+    unicode = str
+
+
+def main(argv=None):
+    if argv is None:
+        argv = sys.argv[1:]
+
+    parser = argparse.ArgumentParser(prog="trustme")
+    parser.add_argument(
+        "-d",
+        "--dir",
+        default=os.getcwd(),
+        help="Directory where certificates and keys are written to. Defaults to cwd.",
+    )
+    parser.add_argument(
+        "-i",
+        "--identities",
+        nargs="*",
+        default=("localhost", "127.0.0.1", "::1"),
+        help="Identities for the certificate. Defaults to 'localhost 127.0.0.1 ::1'.",
+    )
+    parser.add_argument(
+        "--common-name",
+        nargs=1,
+        default=None,
+        help="Also sets the deprecated 'commonName' field (only for the first identity passed).",
+    )
+    parser.add_argument(
+        "--key-size",
+        type=int,
+        default=2048,
+        help="Key size of the certificate generated. Defaults to 2048.",
+    )
+    parser.add_argument(
+        "-q",
+        "--quiet",
+        action="store_true",
+        help="Doesn't print out helpful information for humans.",
+    )
+
+    args = parser.parse_args(argv)
+    cert_dir = args.dir
+    identities = [unicode(identity) for identity in args.identities]
+    common_name = unicode(args.common_name[0]) if args.common_name else None
+    key_size = args.key_size
+    quiet = args.quiet
+
+    if not os.path.isdir(cert_dir):
+        raise ValueError("--dir={} is not a directory".format(cert_dir))
+    if len(identities) < 1:
+        raise ValueError("Must include at least one identity")
+
+    # Generate the CA certificate
+    trustme._KEY_SIZE = key_size
```

Comment:
This is likely to break things in the future. For example, if the last cli tests uses a 4096 key size, then all other tests will use a 4096 key size too. `_KEY_SIZE` should become `_DEFAULT_KEY_SIZE` and `issue_cert` should accept a `key_size` parameter (but it needs to be tested).